exercism/wordy/wordy.js

```diff
@@ -0,0 +1,39 @@
+module.exports = {
+  WordProblem: class WordProblem {
+    constructor(input){
+      this.input = input,
+      this.operands = {
+        'plus': '+',
+        'minus': '-',
+        'multiplied': '*',
+        'divided': '/'
+      },
+      this.sequence = []
+
+    }
+
+    answer() {
+      let splitString = this.input.replace('?',"")
+      .replace(/by/g,'').split(' ').slice(2)
+      let operandsKeys = Object.keys(this.operands)
+      let match = false
+      operandsKeys.forEach(operand => {
+        if (splitString.includes(operand)) {
+          match = true
+        }
+      })
+      if (!match) {
+        throw new Error()
```

Comment:
Consider adding a meaningful error message to the `Error` to provide additional information to anyone who encounters this error:
```js
  throw new Error( "No operators were found in the input string." )
```